master/internal/core.go

```diff
@@ -55,6 +55,7 @@ type Master struct {
 	rwCoordinator *actor.Ref
 	provisioner   *actor.Ref
 	db            *db.PgDB
+	proxy         *actor.Ref
```

Comment:
question: do we need to make this class variable?

Reply:
It looks fine for now since we put other components in it. We probably should pass the master stuct to all places rather than using the address to find the actor, which is done in `command.go`